impls/ntt-data-on-premise.yaml

```diff
@@ -0,0 +1,165 @@
+name: ntt-data-on-premise
+description: https://github.com/Green-Software-Foundation/sci-guide/blob/dev/use-case-submissions/nttdatta-On-Premise-Web-system.md
+tags:
+  kind: web
+  complexity: moderate
+  category: on-premise
+config:
+  pipeline:
+    calculation: ntt-web
+    normalization: default
+    aggregation: default
+    enrichment: default
+# Single observation
+graph:
+  layer-3-switch: # an advanced grouping node
+    model: default  
+    config:
+    children: 
+      switch-1:
+        common:
+            p_baseline: 92.5 # W Baseline power based on full port 100% traffic for Cisco Catalyst 9300
```

Comment:
Speaking to my previous point I think we have 3 models here.

- A new energy model for network switches: that is using a custom equation for calculating the energy for a network interface, takes as input baseline and observations of input/output rates.
- PUE model: Somehow scales the energy value (so there is a hierarchy of models?)
- sci embodied model, same as all the previous use cases.